authentication/utils.py

```diff
@@ -96,3 +96,42 @@ def get_photo_url(cls, uid):
         """
         # pylint: disable=protected-access
         return auth.get_user(uid)._data['photoUrl']
+
+    @classmethod
+    def send_message(cls, data):
+        """
+        Gets the message content
+        """
+        club = data['club']
+        topic = 'C_' + str(club.id)
+        print(topic)
+        msg_notification = messaging.Notification(
+            title="New Workshop in "+str(club.name),
+            body=data['title']+" on "+str(data['date'].strftime('%d-%m-%Y')),
+            image=data.get('image_url',''))
+        message = messaging.Message(
+            notification=msg_notification,
+            topic=topic
+        )
+
+        response = messaging.send(message)
+        print('Successfully sent message:', response)
```

Comment:
Just a small suggestion, you should use python's `logging` module to log messages rather than print. Print works well with `runserver` command but it won't be shown in production logs.